operators/gitlab-operator-kubernetes/0.6.3/manifests/gitlab-operator-kubernetes.clusterserviceversion.yaml

```diff
@@ -0,0 +1,705 @@
+              resourceNames:
+                - gitlab-nginx-ingress-scc
+              resources:
+                - securitycontextconstraints
+              verbs:
+                - use
+            - apiGroups:
+                - ""
+              resources:
+                - namespaces
+              verbs:
+                - get
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+                - pods
+                - secrets
+                - endpoints
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - ""
+              resources:
+                - services
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - networking.k8s.io
+              resources:
+                - ingresses
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - networking.k8s.io
+              resources:
+                - ingresses/status
+              verbs:
+                - update
+            - apiGroups:
+                - networking.k8s.io
+              resources:
+                - ingressclasses
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+              verbs:
+                - get
+                - update
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+              verbs:
+                - create
+            - apiGroups:
+                - ""
+              resources:
+                - events
+              verbs:
+                - create
+                - patch
+          serviceAccountName: gitlab-nginx-ingress
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: false
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+  keywords:
+    - GitLab
+    - DevOps
+    - CI/CD
+    - DAST
+    - SAST
+  links:
+    - name: GitLab Docs
+      url: https://docs.gitlab.com/
+    - name: GitLab Operator Documentation
+      url: https://docs.gitlab.com/charts/installation/operator.html
+    - name: GitLab and Kubernetes
+      url: https://about.gitlab.com/solutions/kubernetes/
+    - name: Gitlab Reference Architecture
+      url: https://docs.gitlab.com/ee/administration/reference_architectures/
+    - name: GitLab Contact Sales
+      url: https://about.gitlab.com/sales/
+  maintainers:
+    - email: distribution@gitlab.com
+      name: GitLab Distribution Team
+  maturity: alpha
+  minKubeVersion: 1.19.0
+  provider:
+    name: GitLab Inc
+    url: https://about.gitlab.com/
+  version: 0.2.0
```

Comment:
Did you mean `  version: 0.6.3`?

Reply:
Thanks for checking that, I wonder if a difference in the version of `operator-sdk` caused some drift here 🤔 Should be fixed now.